generators/app/index.js

```diff
@@ -273,6 +273,20 @@ module.exports = JhipsterGenerator.extend({
                 local: require.resolve('../client')
             });
 
+        },
+
+        composeLanguages: function () {
```

Comment:
ok just noticed that it writes some server side files as well so its should be fine here I guess. but we need to document that when generating a client only langues needs to be added manually or handle that somehow